### PR TITLE
Add Cerbere recipe

### PR DESCRIPTION
**Pull request by Marie59**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Marie59** [comment]
Hi @conda-forge/help-python this recipe is ready !

File: recipes/cerbere/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  name: cerbere
+  version: {{ version }}
+
+source:
+  url: https://gitlab.ifremer.fr/cerbere/cerbere/-/archive/V2.1.0/cerbere-{{ version }}.tar.gz
+  sha256: de3aae083cca91381b642a10282c9718849b2a46beda596c67cb8eb8ceb7ebee
+
+build:
+  number: 0
+  skip: true
```
Comment (dopplershift) [inline recipes/cerbere/meta.yaml:13]: This is skipping all builds. I think this is pure Python and could be noarch?
```suggestion
  noarch: python
```

File: recipes/cerbere/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+requirements:
+  host:
+    - python
+    - pip
+    - wheel
+
+  run:
+    - python
```
Comment (dopplershift) [inline recipes/cerbere/meta.yaml:22]: For noarch, need to put Python version range here:
```suggestion
    - python >=3.6
    - pip
    - wheel

  run:
    - python >=3.6
```

File: recipes/cerbere/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    - pyyaml >=5.3
+    - netcdf4 >=1.5
+    - scipy >=1.5
+    - shapely >=1.7
+    - xarray  >=2022.3.0
```
Comment (dopplershift) [inline recipes/cerbere/meta.yaml:27]: Minor lint:
```suggestion
    - xarray >=2022.3.0 
```

**Review by dopplershift — CHANGES_REQUESTED** [review]

**Comment by Marie59** [comment]
Thank you for your help @dopplershift !

**Comment by Marie59** [comment]
@conda-forge/help-python I don't understand how to fix the issue with poetry_dynamic_versioning
Could you give me a hint ?

File: recipes/cerbere/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+    - wheel
+    - poetry_dynamic_versioning
```
Comment (BastianZim) [inline recipes/cerbere/meta.yaml:22]:
```suggestion
    - poetry-dynamic-versioning
```

Comment (Marie59) [reply recipes/cerbere/meta.yaml:22]: Ooh duh thank you !

**Pull request closed without merging**